packages/tokens/src/Icons.ts

```diff
@@ -0,0 +1,2027 @@
+export const icons = {
```

Comment:
As discussed it would be nice to see if we can put all the svg files in a folder and maybe have a typed function to get the content of a file?

It would be nice to be able to import React components with something similar to: `import ZoomOut from '@coveord/plasma-tokens/svg/zoomOut/size16px.svg';` using a loader like https://www.npmjs.com/package/@svgr/webpack or the equivalent with other bundlers

Reply:
I'm also not sure this file could be tree shaken if we only need one svg

Reply:
Done